test/parallel/test-fs-write-stream-patch-open.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const common = require('../common');
+const fs = require('fs');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+common.expectWarning(
+  'DeprecationWarning',
+  'WriteStream.prototype.open is deprecated', 'DEP0XXX');
+const s = fs.createWriteStream(`${tmpdir.path}/out`).on('error', () => {});
```

Comment:
Why the error-swallowing? (Sorry if there's an obvious answer and I'm just being clueless.)

Reply:
because we just want an instance, we don't care whether the file is actually opened or not.